tools/templates/DeviceBindingTemplate/dotnet_new_device-binding_csharp/_DeviceBinding.cs

```diff
@@ -9,12 +9,19 @@
 
 namespace Iot.Device._DeviceBinding
 {
+    /// <summary>
+    /// Add documentation here
```

Comment:
I'm fine with keeping this, but just wondering if it would actually be better to remove them so that people get compiler errors when they don't have documentation. If we add these they could potentially just leave this and we may not catch it in review. That said, I would also like our template to generate code that can compile, so perhaps keeping it is right :). I'll let you be the one to decide here.

Reply:
I was exactly thinking the same, but the compiler beat me to it: The build fails without these changes (this somehow also gets compiled)